electron/main/index.ts

```diff
@@ -13,7 +14,8 @@ import { update } from "./update";
 import { runBackend } from "./backend";
 import { saveNodePack } from "./node-pack-save";
 import { killSubProcess } from "./cmd";
-import fs from "fs";
+import { writeFileSync } from "fs";
```

Comment:
```suggestion
import fs from "fs";
```
`handleWriteFileSync` has an error otherwise